addons/stock/models/product.py

```diff
@@ -76,7 +76,7 @@ class Product(models.Model):
     reordering_min_qty = fields.Float(compute='_compute_nbr_reordering_rules')
     reordering_max_qty = fields.Float(compute='_compute_nbr_reordering_rules')
 
-    @api.depends('stock_quant_ids', 'stock_move_ids')
+    @api.depends('stock_move_ids.product_qty', 'stock_move_ids.state')
     def _compute_quantities(self):
```

Comment:
note: why this change: no need to check on both quants and moves, and stock.move will be used later in the method to compute the theoretical quantities, so just keep the stock move dependency